stable_baselines3/common/logger.py

```diff
@@ -159,12 +159,12 @@ class HumanOutputFormat(KVWriter, SeqWriter):
         no longer than 79 characters wide.
     """
 
-    def __init__(self, filename_or_file: Union[str, TextIO], max_length: int = 36):
+    def __init__(self, filename_or_file: Union[str, TextIOBase], max_length: int = 36):
```

Comment:
Looking at https://stackoverflow.com/questions/61000227/type-checking-issue-with-io-textiobase-in-a-union and https://github.com/python/typeshed/pull/10135/files, why not `TextIO` for the annotation
and `TextIOBase` for the check?

```python
import io
import sys
import typing


def test(out: typing.TextIO) -> None:
    assert isinstance(out, io.TextIOBase)
    out.write("test")


test(sys.stdout)

file = open("test.text", "w")

test(file)
```

Reply:
Let's try this (so unintuitive...)